src/components/utils/src/logger.cc

```diff
@@ -53,6 +53,7 @@ void deinit_logger() {
   logger::logger_status = logger::LoggerThreadNotCreated;
 }
 
+// cppcheck-suppress unusedFunction
```

Comment:
```suggestion
// cppcheck-suppress unusedFunction // used by LOG_WITH_LEVEL
```

Reply:
Done in 9f1a645791e598169044d384c2a131addb74c075